stat/init.lua

```diff
@@ -218,6 +219,27 @@ local function stat(args)
     stats['statperf.fibers'] = ed_cpu - st_cpu
     stats['statperf.fibers_t'] = ed_time - st_time
 
+    if detailed_replication then
+        stats['stat.detailed_replication'] = {
+            my_host = util.get_my_replica_host(),
+            replicaset = {},
+        }
+        for _, v in ipairs(box.info.replication) do
+            local item = {
+                uuid = v.uuid,
+                lsn = v.lsn,
+                host = stats['stat.detailed_replication']['my_host'],
+            }
+            if v.upstream ~= nil then
+                item.host = util.get_replica_host(v.upstream.peer)
```

Comment:
I seems that the value will incorrect when we observe a master from a replica.

Reply:
More on that: we show upstream host:port here, but the field named as 'host', so I would interpret it as listen host:port,